cloud/satellite.go

```diff
@@ -298,3 +308,16 @@ func satelliteStatus(status pb.SatelliteStatus) string {
 		return SatelliteStatusUnknown
 	}
 }
+
+var validSizes = map[string]bool{
+	"":                  true,
```

Comment:
I left empty-string as supported here because in theory the server accepts it and defaults to large in that case. The outer command code defaults it to medium now though 🤷